Use parseEnv instead of dotenv and avoid side effects (#1947)
changesets/changesets: .changeset/late-animals-drive.md

---
"@changesets/changelog-github": patch
---

> Use `loadEnvFile` instead of `dotenv` to load the `.env` file

Reviewer: ```suggestion
Replaced `dotenv` with `loadEnvFile` for loading `.env` files
```

i think generally changesets should use past tense, as they are a log of what _has changed_ in a package, compared to commit descriptions that use present tense as they describe what a commit _does_ or _will do_ to a repository

Author: I've always used present tense here as they work like git commits to me. It depends how you read it but they could also describe what a specific version does. I don't think we need to standardize the format for now though.

Reviewer: it would be nice to have one before the v3 release to make it look nicer though :)

Reviewer 2: we should go through the final changesets/changelog updates manually before releasing final v3 anyway, so we should be able to adjust the lanaguage then
